My_Calc_Project.ipynb

```diff
@@ -102,7 +102,7 @@
         "import random\n",
         "\n",
         "class calculator:\n",
-        "    def __init__(self, name='', country=''): #this is just an intro to get to know user\n",
+        "    def __init__(self, name='', country=''):\n",
```

Comment:
I think you have an error here, you didn't completely remove all the comments. See `\n"`